### PR TITLE
Create orders from dashboard

### LINKED ISSUE #1125: Add ability to create order from dashboard
Staff users should be able to create orders from the dashboard. This will consist of the following parts:
- [ ] Adding products
- [ ] Ability to apply discounts or vouchers
- [ ] Setting shipping method and price
- [ ] Assigning a customer
- [ ] Setting shipping and billing address
- [ ] Sending confirmation email to the customer once an order is created. Since confirmation email contains a link to an order page, customers should be able to use it to pay for an order themselves.

Related issue: #1908 - on the order details page we should also have the ability to mark an order as paid. This will help when staff is importing an old order or the payment was received in some other way.

### PR DESCRIPTION
**Pull request by mad-anne**
closes #1125

### CODE REVIEW

File: saleor/dashboard/order/forms.py [hunk 1]
```diff
@@ -307,3 +310,29 @@ class OrderFilterForm(forms.Form):
...
+            reverse('dashboard:ajax-users-list'))
+
+    def save(self, commit=True):
+        user = self.instance.user
+        self.instance.user_email = user.email
+        self.instance.shipping_address = user.default_shipping_address
+        billing_address = user.default_billing_address
+        if not billing_address:
+            billing_address = Address.objects.create(
+                country=settings.DEFAULT_COUNTRY)
```
Comment (Pacu2) [inline saleor/dashboard/order/forms.py]: Should we pass user's first name and last name here as well?

Comment (patrys) [reply saleor/dashboard/order/forms.py]: That does not look right. How useful is an empty address object?

Comment (mad-anne) [reply saleor/dashboard/order/forms.py]: You are both right. I think we could allow from now to store null as billing_address (for purpose of creating orders from dashboard).

Comment (patrys) [reply saleor/dashboard/order/forms.py]: Right, I see two options here: either ask the admin to provide a valid address right away or allow orders to have missing addresses. The second is easier to implement but we should make sure that all actions like payment and shipping are blocked until a valid address is provided.

**Comment by elwoodxblues** [comment]
We should hold this PR for now. First we have to merge #1194 as it adds widgets and logic that may be useful here. Then we should rethink the requirements more carefully. The logic behind order management is getting more and more complex and I'm afraid that if we don't clean up the code now, it'll become difficult to maintain.

**Pull request closed without merging**